From: Cole Robinson <crobinso@redhat.com>
Date: Fri, 27 Oct 2017 09:42:54 +0200
Subject: [PATCH virt-manager] devicepanic: Don't return empty model list (bz
 #1505532)

Callers don't expect it, and it breaks opening the addhardware dialog
for aarch64 VMs

https://bugzilla.redhat.com/show_bug.cgi?id=1505532
(cherry picked from commit f7c8cf9f667f2b0122f5689009d74c94c38c7316)
---
 virtinst/devicepanic.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/virtinst/devicepanic.py b/virtinst/devicepanic.py
index 63a88f71..9f44e93e 100644
--- a/virtinst/devicepanic.py
+++ b/virtinst/devicepanic.py
@@ -49,12 +49,13 @@ class VirtualPanicDevice(VirtualDevice):
     @staticmethod
     def get_models(os):
         if os.is_x86():
-            return [VirtualPanicDevice.MODEL_ISA, VirtualPanicDevice.MODEL_HYPERV]
+            return [VirtualPanicDevice.MODEL_ISA,
+                    VirtualPanicDevice.MODEL_HYPERV]
         elif os.is_pseries():
             return [VirtualPanicDevice.MODEL_PSERIES]
         elif os.is_s390x():
             return [VirtualPanicDevice.MODEL_S390]
-        return None
+        return []
 
     @staticmethod
     def get_default_model(os):
